src/utils/viewer/viewer.cpp

```diff
@@ -210,17 +224,21 @@ Viewer::Viewer(const Arguments& arguments)
   renderCamera_->node().setTransformation(
       rgbSensorNode_->absoluteTransformation());
 
-  timeline_.start();
+  // fonts
+  textRenderer_.reset(new gfx::TextRenderer(
+      Mn::Vector2(Mn::GL::defaultFramebuffer.viewport().size()).aspectRatio()));
```

Comment:
```suggestion
      Mn::Vector2(framebufferSize()).aspectRatio()));
```

(I still need to update all docs and example code to use this shorter variant, sorry.)

Reply:
Somehow I found it from your docs.
Will change.

Reply:
Yeah, that's what I'm saying :) most of the occurences where this is used was written *before* `framebufferSize()` was a thing, so I need to update all those docs.